cmd/mayactl/app/command/volume.go

```diff
@@ -24,23 +24,36 @@ import (
 
 var (
 	volumeCommandHelpText = `
-	Usage: maya volume <subcommand> [options] [args]
+The following command provides operations related to a Volume.
```

Comment:
... commands allow you to operate on a Volume such as read, list, and so on.

Reply:
done